CITATION.bib

```diff
@@ -1,6 +1,8 @@
-@misc{Qiskit,
-    author = {{Qiskit contributors}},
-    title = {Qiskit: An Open-source Framework for Quantum Computing},
-    year = {2023},
-    doi = {10.5281/zenodo.2573505}
+@misc{javadiabhari2024quantum,
+      title={Quantum computing with Qiskit},
+      author={Ali Javadi-Abhari and Matthew Treinish and Kevin Krsulich and Christopher J. Wood and Jake Lishman and Julien Gacon and Simon Martiel and Paul D. Nation and Lev S. Bishop and Andrew W. Cross and Blake R. Johnson and Jay M. Gambetta},
```

Comment:
I don't remember to what degree it matters (might depend on the `.bst` file), but BibTeX typically prefers `Lastname, Firstnames` format:
```suggestion
      author={Javadi-Abhari, Ali and Treinish, Matthew and Krsulich, Kevin and Wood, Christopher J. and Lishman, Jake and Gacon, Julien and Martiel, Simon and Nation, Paul D. and Bishop, Lev S. and Cross, Andrew W. and Johnson, Blake R. and Gambetta, Jay M.
},
```

@levbishop: do you remember if it actually matters / in what circumstances?

Reply:
FWIW, I literally just copied this bibtex from what the "export BibTeX citation" button on the arxiv page generated.

Reply:
For future reference the source code that generated this is here: https://github.com/arXiv/arxiv-browse/blob/97eeef6ab8db5833141c69c055ef08cd05a0b6e7/browse/formatting/cite.py#L11 if we have concrete improvements to the formatting we should open a PR proposing them over there.

Reply:
(Copied from slack)

> I would imagine that they very specifically don’t want to play with the author lists there
>
> the reason for the explicit comma form being preferable is (iirc) because it’s basically impossible to know which names should be considered "first" and which "last"